ingest/scripts/call_loculus.py

```diff
@@ -171,6 +173,65 @@ def get_or_create_group_and_return_group_id(config: Config, allow_creation: bool
     return create_group_and_return_group_id(config)
 
 
+def filter_fasta_by_submission_ids(
+    fasta_dict: dict[str, Any], submission_ids: list[str], config: Config
+):
+    if config.segmented:
+        segmented_submission_ids = [
+            f"{submission_id}_{segment}"
+            for submission_id in submission_ids
+            for segment in config.nucleotide_sequences
+        ]
+        return [
+            fasta_dict[submission_id]
+            for submission_id in segmented_submission_ids
+            if submission_id in fasta_dict
+        ]
+    return [
+        fasta_dict[submission_id] for submission_id in submission_ids if submission_id in fasta_dict
+    ]
+
+
+def post_fasta_batches(
+    url,
+    fasta_file: str,
+    metadata_file: str,
+    config: Config,
+    params: dict[str, str],
+    chunk_size=60000,
+) -> requests.Response:
+    """Chunks metadata files, joins with sequences and submits each chunk via POST."""
+    df = pd.read_csv(metadata_file, sep="\t")
+    logger.info(df.columns)
+    sequences_dict = {record.id: record for record in SeqIO.parse(fasta_file, "fasta")}
+    submission_ids = df["submissionId"].tolist()
+    for batch_num, submission_id_chunk in (
```

Comment:
`batch_num` sounds like it'd be a the batch number (0, 1, 2, 3, ...) but it's actually (0, 60000, ...)

Another variable name would be good, but I also can't think of one so feel free to ignore.